dev-packages/application-manager/package-lock.json

```diff
@@ -1,12 +1,641 @@
 {
```

Comment:
You should use yarn, not npm. Please delete all npm lock files.

Reply:
Removed in my tree already :wink:

Reply:
but it is still here

Reply:
Weird, I already removed it in my tree fsr it didn’t stage :/